src/app/components/Copilot/Copilot.tsx

```diff
@@ -136,27 +139,24 @@ export function CopilotChat(): React.JSX.Element {
   };
     
   // for setting the initial copilot chat that takes place on page load.
+  const getInitialChat = async () => {
+    await new Pieces.ConversationsApi().conversationsSnapshot({}).then(output => {
+      const previousChats = output.iterable.map(conversation => conversation.name);
+      setChats(previousChats);
+      const _name = previousChats[0];
+      setChatSelected(_name);
+      GlobalConversationID = output.iterable.at(0).id;
+    });
+  };
+
+  // Function to handle chat selection
+  const handleChatSelection = (index: number) => {
+    setCurrentChatIndex(index);
+    setChatSelected(chats[index]);
```

Comment:
it looks like this is working properly, but we need to do a little bit of work to the UI so that all of the other chats are are not showing with buttons as well. let me know what you are able to find 😄

Reply:
OK,I'll look into that

Reply:
<div className="messages">
        {/* Render selected chat content with buttons */}
        {currentChatIndex !== -1 && (
          <div>
            <p>{message}</p>
            <button>Button 1</button>
            <button>Button 2</button>
            {/* Add more buttons as needed */}
          </div>
        )}
      </div>
    </div>
  </div>
);          @jordan-pieces can you take a look at this code will it work fine, actually I'm that good with UI part